### PR TITLE
Update project to use juggler@3

### PR DESCRIPTION
**Pull request by gunjpan**
Update package.json to use
loopback-datasource-juggler@3.x

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -42,7 +42,7 @@
     "loopback": "^2.22.0",
     "loopback-boot": "^2.6.5",
     "loopback-component-explorer": "^2.1.0",
-    "loopback-datasource-juggler": "^2.39.0",
+    "loopback-datasource-juggler": "^3.0.0",
```
Comment (bajtos) [inline package.json:45]: I am afraid you are changing a wrong place. You should remove this line entirely and use `loopback: '^3.0.0'` instead, see L42.

**Review by bajtos — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (gunjpan) [reply package.json:45]: @bajtos : Yes, I noticed that and corrected it on Local. But facing failures, trying to resolve them and didn't push yet.

**Comment by bajtos** [comment]
@gunjpan ping, what's the status of this pull request?

**Comment by gunjpan** [comment]
@bajtos : No progress ATM as focused on getting `geopoint` out of the door. It's committed for this sprint, aiming to start on it today/tomorrow the earliest.

**Comment by bajtos** [comment]
I think this should be re-assigned to @strongloop/squad-apex ?

**Comment by gunjpan** [comment]
@bajtos : I've committed to working on completing it this sprint. No need to reassign, but if @strongloop/squad-apex want to take it, I'm ok with it too. Thanks.

**Pull request closed without merging**